crates/typst-library/src/foundations/func.rs

```diff
@@ -16,20 +16,18 @@ use crate::foundations::{
     PluginFunc, Scope, Selector, Type, Value,
 };
 
-/// A mapping from argument values to a return value.
+/// 引数値から戻り値への写像。
 ///
-/// You can call a function by writing a comma-separated list of function
-/// _arguments_ enclosed in parentheses directly after the function name.
-/// Additionally, you can pass any number of trailing content blocks arguments
-/// to a function _after_ the normal argument list. If the normal argument list
-/// would become empty, it can be omitted. Typst supports positional and named
-/// arguments. The former are identified by position and type, while the latter
-/// are written as `name: value`.
+/// 関数名の直後に括弧で囲まれたカンマ区切りの関数の _引数_ のリストを書くことにより関数を呼び出すことができます。
+/// 加えて、通常の引数リストの後に任意の数のコンテンツブロック引数を関数に渡すことができます。
```

Comment:
前文の方法だけでなくこの文の方法でもできるというニュアンスを出してみてはいかがでしょうか？「加えて」でできていると思いますが、助詞も使うとよりできると思います。
```suggestion
/// 加えて、通常の引数リストの後に任意の数のコンテンツブロック引数を関数に渡すこともできます。
```